src/buzz/buzzparser.c

```diff
@@ -606,6 +606,8 @@ int parse_if(buzzparser_t par) {
    /* Jamp to label 1 if the condition is false */
    /* Label 1 is either if end (in case of no else branch) or else branch */
    chunk_append("\tjumpz " LABELREF "%u", lab1);
+   // Allow the use of non-cuddled brackets 
```

Comment:
Please avoid the C++ comment style. Use `/* comment */` instead of `// comment`.